app/i18n/en/sub.php

```diff
@@ -68,7 +68,7 @@
 		'error' => 'This feed has encountered a problem. Please verify that it is always reachable then update it.',
 		'filteractions' => array(
 			'_' => 'Filter actions',
-			'help' => 'Write one search filter per line.',
+			'help' => 'Write one search filter per line. Operators <a href="https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field" target="blank">see documentation</a>.',
```

Comment:
```suggestion
			'help' => 'Write one search filter per line. Operators <a href="https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field" target="_blank">see documentation</a>.',
```

And the same for all other translations

Reply:
oh no :/

Reply:
fixed